packages/react-examples/src/react-charting/DonutChart/DonutChart.Dynamic.Example.tsx

```diff
@@ -43,12 +48,25 @@ export class DonutChartDynamicExample extends React.Component<IDonutChartProps,
     };
     return (
       <div>
+        <Checkbox
+          label="Hide values"
+          checked={this.state.hideLabels}
+          onChange={this._onHideCheckChange}
```

Comment:
rename onChange functions for better meaning.

Reply:
done